### ISSUE TITLE
[Bug]: Upcoming videos/Premieres not showing any date or time info in Subscriptions list

### ISSUE THREAD

**Issue by HearthstoneBugs** [FreeTubeApp/FreeTube#4147]
### Guidelines

- [X] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [X] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [X] This issue contains only one bug.

### Describe the bug

In the Subscriptions list, any Upcoming Premieres don't show any date or time info.

On YouTube's Subscriptions list you'll see data showing:

Title of video
'Channel Name' Premieres 10/13/23, 21:00 (for example)

But FreeTube shows none of that on the Subs page.

### Expected Behavior

On the Subscriptions page expect to see:

'Channel Name' Premieres 10/13/23, 21:00 (for example)

### Issue Labels

visual bug

### FreeTube Version

v0.19.1 Beta

### Operating System Version

Windows 10 Pro 22H2

### Installation Method

.exe

### Primary API used

Local API

### Nightly Build

- [ ] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by HearthstoneBugs** [comment]
Thanks for your reply.

I've not deliberately enabled RSS, but I have noticed that due to my large subscriptions list it's forcing RSS to avoid rate limiting, so as a test I've just deleted all of my subs (exported them first of course) and just subbed to a channel for a forthcoming premiere and that's now listed correctly.

So it looks like I need to reduce my subs list to 125 or less (I think 125 is the maximum?).

A bit of a nuisance really as I do sub to a lot of channels, unless there's another way around reducing the subs list but still being about to see Premiere details, etc in the list?
